tests/shared/src/test/scala/org/typelevel/toolkit/ScalaCliProcess.scala

```diff
@@ -69,10 +69,9 @@ object ScalaCliProcess {
       .evalTap { path =>
         val header = List(
           s"//> using scala ${BuildInfo.scalaBinaryVersion}",
-          s"//> using toolkit typelevel:${BuildInfo.version}",
-          s"//> using platform ${BuildInfo.platform}"
-        ).mkString("", "\n", "\n")
-        Stream(header, scriptBody.stripMargin)
+          s"//> using toolkit typelevel:${BuildInfo.version}"
+        ) ::: platformSpecificDirectives
```

Comment:
Instead of splitting sources (now requires to read three files in different directories to understand what is happening) can we just write this as a conditional on `BuildInfo.platform)` (only requires to read three consecutive lines to understand)?

Reply:
hmm, yes, it's leaner

Reply:
No wait, with the current build.sbt `BuildInfo.nativeVersion` is visible only under `/native`, so I can't add the if here unless I move the definition of `BuildInfo.nativeVersion` in the main/agnostic preoject settings.

Reply:
At the end of the day, there's no harm in making `nativeVersion` visible to all the platforms, that's why the `native` header is there. I simplified the PR.